idea-plugin/src/test/kotlin/arrow/meta/ide/phases/resolve/QuoteSystemCacheTest.kt

```diff
@@ -14,9 +16,7 @@ class QuoteSystemCacheTest : LightPlatformCodeInsightFixture4TestCase() {
   fun testIncrementalCacheUpdate() {
     val file = myFixture.addFileToProject("testArrow/source.kt", "package testArrow")
 
-    // this is the initial rebuild and the initial cache population
-    val cache = QuoteSystemCache.getInstance(project)
-    cache.forceRebuild()
+    project.testQuoteSystem()?.forceRebuild(project)
```

Comment:
could non-null be enforced? There would be something quite wrong if the quote system isn't available in the test.

Reply:
If the service is not available the test will automatically fail. But we can throw a refined Error message without a stacktrace in that case.